dist/web3-light.js

```diff
@@ -13192,8 +13352,11 @@ module.exports = BigNumber; // jshint ignore:line
 
 },{}],"web3":[function(require,module,exports){
 var web3 = require('./lib/web3');
+
 web3.providers.HttpProvider = require('./lib/web3/httpprovider');
 web3.providers.QtSyncProvider = require('./lib/web3/qtsync');
+web3.providers.IpcProvider = require('./lib/web3/ipcprovider');
```

Comment:
But mist is using electron. Electron is a subset of node. You can't do [`ipc`](https://en.wikipedia.org/wiki/Inter-process_communication) in normal browser (like chromium). It should be wrapped in the `if` statement.